pkg/api/steve/catalog/types/rest.go

```diff
@@ -52,26 +56,32 @@ type ChartInfo struct {
 	Chart     v3.MapStringInterface `json:"chart,omitempty"`
 }
 
+// ChartUninstallAction represents the input received when uninstalling a chart
 type ChartUninstallAction struct {
-	DisableHooks bool             `json:"noHooks,omitempty"`
-	DryRun       bool             `json:"dryRun,omitempty"`
-	KeepHistory  bool             `json:"keepHistory,omitempty"`
-	Timeout      *metav1.Duration `json:"timeout,omitempty"`
-	Description  string           `json:"description,omitempty"`
+	DisableHooks           bool                `json:"noHooks,omitempty"`
+	DryRun                 bool                `json:"dryRun,omitempty"`
+	KeepHistory            bool                `json:"keepHistory,omitempty"`
+	Timeout                *metav1.Duration    `json:"timeout,omitempty"`
+	Description            string              `json:"description,omitempty"`
+	OperationTolerations   []corev1.Toleration `json:"operationTolerations,omitempty"`
+	AutomaticCPTolerations bool                `json:"automaticCPTolerations,omitempty"`
```

Comment:
not blocking since the existing vars didn't follow this but, are we still following the naming convention for booleans in go? i.e. `isAutomaticCPToleration` or `hasAutomaticCPTolreation`

Reply:
Great point, we're still following that in the Shepherd PRs, but this is not something we can change, this is how REST API changes happen in the linked PR